ros/api/v1/hosts.py

```diff
@@ -197,9 +197,10 @@ def build_system_filters():
             modified_states = []
             for state in states:
                 state = state.capitalize()
-                modified_states.append(state)
                 if state not in SYSTEM_STATES_EXCEPT_EMPTY:
                     abort(400, message='values are not matching')
+                else:
+                    modified_states.append(state)
```

Comment:
nitpick - `else` not necessary 

```suggestion
                if state not in SYSTEM_STATES_EXCEPT_EMPTY:
                    abort(400, message='values are not matching')
                modified_states.append(state)
```